insights/api/C/libinsightstest.go

```diff
@@ -1,7 +1,7 @@
 // main is the package for the C API.
 package main
 
-// most of this is copied from libinsights.go, keep them up to date.
```

Comment:
Little replace all error.

I think `server` is a good name, I think `services` might also work